internal/config/config.go

```diff
@@ -5,7 +5,7 @@ import (
 	"os"
 )
 
-// legacy thing - used only for back-compatiblity command-line tools pg*
+// legacy thing - used only for back-compatiblity command-line tools pg*.
```

Comment:
all `dot` changes better to rollback, we minimise  aestetic changes: https://github.com/doublecloud/transfer/blob/main/CONTRIBUTING.md#aesthetic-changes it's brings little value and pollute blame of a file.